pom.xml

```diff
@@ -893,37 +745,46 @@
 
             </executions>
           </plugin>
-	        <plugin>
-	            <groupId>org.apache.maven.plugins</groupId>
-	            <artifactId>maven-dependency-plugin</artifactId>
-	            <dependencies>
-	                <dependency>
-	                    <groupId>org.apache.felix</groupId>
-	                    <artifactId>maven-bundle-plugin</artifactId>
-	                    <version>2.4.0</version>
-	                    <type>maven-plugin</type>
-	                </dependency>
-	            </dependencies>
-	            <extensions>true</extensions>
-	        </plugin>
         </plugins>
       </build>
     </profile>
 
   </profiles>
 
+  <reporting>
+    <outputDirectory>${project.build.directory}/site</outputDirectory>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-project-info-reports-plugin</artifactId>
+        <version>3.2.2</version>
```

Comment:
See if you can use version `3.0.0` as it doesn't bring in dependencies with CVEs at this time.